MWL Aggregations Deprecation (#28088)
DataDog/documentation: content/en/metrics/guide/custom_metrics_governance.md

{{< img src="/metrics/guide/custom_metrics_governance/metrics_rbac_audit_trail_query.png" alt="Event Explorer with the metrics audit trail query" style="width:90%;" >}}

### Reduce costs from unqueried and unused metrics

To ensure you're not removing valuable visibility while reducing costs, you need to differentiate between the actively queried metrics that your team relies on from the metrics that aren't queried anywhere within the Datadog platform or through the API. Datadog's intelligent query insights continuously computes and analyzes all users' interactions (in Datadog or through the API) on any metric to help identify less valuable, unused metrics.

> Additionally, to reduce costs without accidentally breaking any assets, you need visbility into all metrics that are unused on your platform. Filtering for all unused metrics with the [Related Assets facet][16] gives you further insight into which metircs are used on your assets such as dashboards, notebooks, monitors, and SLOs, allowing you to make informed decisions about your metric configurations. 

Reviewer: ```suggestion
Additionally, to reduce costs without accidentally breaking any assets, you need visbility into all metrics that are unused on your platform. Filtering for all unused metrics with the [Related Assets facet][16] gives you further insight into which metrics are used on your assets, such as dashboards, notebooks, monitors, and SLOs. This can help you make informed decisions about your metric configurations. 
```

#### Identify all unqueried and unused metrics

> Additionally, to reduce costs without accidentally breaking any assets, you need visbility into all metrics that are unused on your platform. Filtering for all unused metrics with the [Related Assets facet][16] gives you further insight into which metrics are used on your assets, such as dashboards, notebooks, monitors, and SLOs. This can help you make informed decisions about your metric configurations. 

Reviewer 2: ```suggestion
Use the [Related Assets facet][16] to gain visibility into the metrics that are unused in Datadog, which can help you reduce costs without accidentally breaking any assets. This facet lets you see which metrics are used on your dashboards, notebooks, monitors, and SLOs, allowing you to make informed decisions about your metric configurations. 
```

Identify your organization's entire list of unqueried and unused metrics:

1. On the [Metrics Summary page][6], find the **Query Activity** facet on the left side. Select the time frame of interest (30, 60, or 90 days).
2. Find the **Related Assets** facet on the left side, and select **Does not have assets**.

> 3. Find the **Configuration** facet on the left side, and select **All Tags**. The combination of these three facets provides you a list of unqueried and unused custom metrics that haven't yet been configured that you can receive immediate cost savings from.

Reviewer 2: ```suggestion
3. Find the **Configuration** facet on the left side, and select **All Tags**. The combination of these three facets provides you a list of any metrics that aren't queried or used on your assets, and which don't already have a custom tag configuration.
```
